optaweb-vehicle-routing-frontend/pom.xml

```diff
@@ -95,6 +116,26 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <artifactId>maven-resources-plugin</artifactId>
```

Comment:
Haven't investigated this yet but I suspect it might be obsolete since it does something with the `build` directory and we're not distributing build artifacts.